test/pytest/test_data/dlrm_bs_1.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
Why is the dlrm test file part of PT 1.12 upgrade PR?

Reply:
I cherry picked the fix for the scriptable tokenizer from the dlrm example branch, will remove these

Reply:
Due to [torchtext backward incompatible](https://github.com/pytorch/text/releases/tag/v0.13.0), [test_example_scriptable_tokenzier.py](https://github.com/pytorch/serve/blob/master/test/pytest/test_example_scriptable_tokenzier.py#L41), which was added a few weeks ago, failed. So @mreso pulled [these fixes](https://github.com/pytorch/serve/pull/1767/commits/ad605276eeaf129ea01222ad3a1fe9bfd9617489) from DLRM.